app/models/concerns/servers/command.rb

```diff
@@ -20,6 +20,23 @@ def execute script
     { status: 'error', output: ex.to_s }
   end
 
+  def execution(execution)
+    script_path = execution.script.copy_to self
+    status      = nil
+
+    Open3.popen2e rails, 'runner', script_path do |stdin, stdout, thread|
+      while line = stdout.gets
+        execution.update_output line
+      end
+
+      status = thread.value.exitstatus.to_i
+    end
+
+    execution.update_output "Exit status: #{status}" unless status == 0
```

Comment:
Para no preguntar dos veces lo mismo, pondría algo así:

```suggestion
    if status == 0
      :success
    else
      execution.update_output "Exit status: #{status}"
      :error
    end
```